src/main/java/in/twizmwaz/cardinal/module/modules/chat/ChatModule.java

```diff
@@ -21,8 +23,11 @@ public void onPlayerChat(AsyncPlayerChatEvent event) {
         event.setCancelled(true);
         //get default channel
         for (TeamChannel channel : GameHandler.getGameHandler().getMatch().getModules().getModules(TeamChannel.class)) {
-            if (channel.getTeam().equals(TeamUtils.getTeamByPlayer(event.getPlayer()))) 
+            if (channel.getTeam().equals(TeamUtils.getTeamByPlayer(event.getPlayer()))) {
                 channel.sendMessage(channel.getTeam().getColor() + "[Team] " + event.getPlayer().getDisplayName() + ChatColor.RESET + ": " + event.getMessage());
+                ConsoleCommandSender console = GameHandler.getGameHandler().getPlugin().getServer().getConsoleSender();
+                console.sendMessage(channel.getTeam().getColor() + "[" + channel.getTeam().getName() + " Team] " + event.getPlayer().getDisplayName() + ChatColor.RESET + ": " + event.getMessage());
```

Comment:
I don't think you need to append "Team," many maps have "Team" as part of the team name, resulting in "[Red Team Team]."